docs/pipeline/TRIGGERING.md

```diff
@@ -0,0 +1,59 @@
+# Modyn Triggering
+
+Alongside the simple triggers, Modyn also provides complex triggers that can be used to trigger the training of a model. 
+
+Despite the line being blurry, complex triggers are generally more sophisticated and require more information to be provided to the trigger - they most of the time cannot be pre-determined via a simple configuration entry but require
+reinjested information from the ongoing pipeline run.
```

Comment:
reingested :D?

Reply:
I wonder why the typo pre-commit hook didn't detect this😅